src/routes/dictionaries/+page.svelte

```diff
@@ -57,27 +56,20 @@
         sequenceAdaptationName,
       );
 
-      if (uploadedDictionaryOrAdaptation === null) {
+      if (uploadedDictionaryOrAdaptation === null || Object.keys(uploadedDictionaryOrAdaptation).length === 0) {
```

Comment:
Does an empty object mean that it failed to upload or is it a slightly different issue that can be relayed specifically?

Reply:
The user may encounter an empty object response if their custom dictionary parser is incorrectly implemented.  We validate that the returned data contains the necessary dictionary or adaptation information. 

We are giving the user more power :P

Reply:
I see. I think it'd be nice to differentiate to the user between if the file successfully uploaded and the format was invalid versus if it just didn't upload because of a network issue or something.

Also, I just noticed this, but would you mind moving the error/success handling to the `uploadDictionaryOrAdaptation` effect function itself?  That would be more consistent with how our requests are handled. We don't typically invoke toast functions unless it's a copy to clipboard action.

Reply:
Sorry it took so long to get to this. I have moved these errors to the `effects.ts`